manifests/charts/istiod-remote/templates/configmap.yaml

```diff
@@ -41,15 +28,15 @@
         {{- end }}
         {{- if $.Values.global.tracer.stackdriver.maxNumberOfAttributes }}
           # The global default max number of attributes per span.
-          maxNumberOfAttributes: {{ $.Values.global.tracer.stackdriver.maxNumberOfAttributes }}
+          maxNumberOfAttributes: {{ $.Values.global.tracer.stackdriver.maxNumberOfAttributes | default "200" }}
```

Comment:
i'm concerned about embedding defaults in the templates. is it possible to avoid this by at least not adding new defaults? otherwise, how will these values be discoverable and syncable with profiles?

Reply:
> is it possible to avoid this by at least not adding new defaults

Not sure what you mean by this? These are not new defaults, just moved. The whole point is these are not synced with the profiles at all, as they are deprecated entirely (not in the profiles at all). This value will be entirely removed in N+1 release.

The root of the issue is that we cannot distinguish between us internally setting a deprecated option and a user setting a deprecated one

Reply:
ok, didn't realize that.